src/NServiceBus.Persistence.TestRunner/NServiceBus.Persistence.TestRunner.csproj

```diff
@@ -14,8 +14,10 @@
     <PackageReference Include="Microsoft.ServiceFabric.FabricTransport.Internal" Version="3.0.467" />
     <PackageReference Include="Microsoft.ServiceFabric.Services" Version="3.0.467" />
     <PackageReference Include="Microsoft.ServiceFabric.Services.Remoting" Version="3.0.467" />
-    <PackageReference Include="NUnit" Version="3.7.1" />
-    <PackageReference Include="NUnitLite" Version="3.7.1" />
+    <PackageReference Include="NUnit" Version="3.11.0" />
+    <PackageReference Include="NUnit3TestAdapter" Version="3.11.2" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
```

Comment:
Same remark as in https://github.com/Particular/NServiceBus.Persistence.ServiceFabric/pull/82#pullrequestreview-187663009